README.md

```diff
@@ -1,2 +1,45 @@
-# strike-off-objections-web
+# Strike Off Objections Web
+
+### Overview
+
 Web frontend for submitting objections to a company being struck off the register.
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [NodeJS](https://nodejs.org/en/)
+- [Homebrew](https://formulae.brew.sh/formula/node)
```

Comment:
I'd argue homebrew is more part of the docker env setup so we don't need to mention it here

Reply:
it's actually a package manager for Mac, you don't need it to run the app.

Reply:
No more homebrew